recovery/recovery.py

```diff
@@ -74,18 +70,32 @@ def __init__(self):
                 'flag': '--file',
                 'help': (
                     'Set name of the file to save data to/load data from. Default name is empty. '
-                    'If name is empty: load operation will use the last backup file, '
+                    'If name is empty: load operation will use the last available backup file (rdb or json), '
```

Comment:
The script only loads json files.

Reply:
Not really, we can specify an rdb backup file as an argument https://github.com/YangCatalog/backend/blob/f4333d52155752cd2915fea724526f85ad5d089b/recovery/recovery.py#L177, so that’s why I’ll better remove this functionality entirely for now

Reply:
Ooh I see. Well this would work for gzipped json files, but we don't create those anywhere, so it would be less confusing to remove it, yeah.